src/api/users/mutations/loginUser/index.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import bcrypt from 'bcrypt'
+import type { ValidationError } from '@types/ValidationError'
+import createToken from '@models/Token/createToken'
+import retrieveToken from '@models/Token/retrieveToken'
+import User from '@models/User'
+import { AUTHENTICATION_ERROR_INVALID_CREDENTIALS } from '@constants/errors'
+
+export type LoginUserArgs = {
+  email: string,
+  password: string
+}
+
+type LoginUserResponse = {
+  token?: string,
+  errors: Array<ValidationError>
+}
+
+const resolver = async (
+  _: any,
+  args: LoginUserArgs
+): Promise<LoginUserResponse> => {
+  const errors: Array<ValidationError> = []
+  const email: string = args.email
+  const password: string = args.password
```

Comment:
Not that it matters too much but:
- You can destructure args directly in the function params instead of doing this.
```js
const resolver = async (
  _: any,
  { email, password }: LoginUserArgs
)
```
-Btw you don’t always need to type variables, they’re automatically inferred in some cases (but well, that can’t hurt)